scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -375,7 +375,7 @@
         </li>
         <li class="list-group-item">
           SPIDEX
-          <span class="float-end">{{ variant.spidex_human }}</span>
+          <span class="float-end">{{ variant.spidex|spidex_human if variant.spidex else none|spidex_human }}</span>
```

Comment:
This code is like this to avoid errors. When variant.spidex is Undefined (jinja2.runtime.Undefined) it gets casted to None.